### DISCUSSION TITLE
How to add annotations with latest release

### DISCUSSION THREAD

**Discussion by smcprae** [pikepdf/pikepdf#637] [Q&A]
I am new to Python and thus pikepdf.

I have tried adding annotations to a page, but can't get it to work. The page.Annots syntax fails, but using page.get('/Annots') allows me to get the dictionary, but I can't see how to set the dictionary back in place on the page. I would expect there to be a page.set(annots_dict) or something like that, but I can't seem to get my head around this.

**Comment by jbarlow83** [comment]
`page['/Annots'] = pdf.make_indirect(Dictionary(...))` should do the job
